### PR TITLE
Noted that search term is required

### PR DESCRIPTION
**Pull request by rosswesleyporter**
Noted that search term is required

### CODE REVIEW

**Review by Jazzagi — COMMENTED** [review]
Looks good 💯

File: vanilla_bot/config.py [hunk 1]
```diff
@@ -6,5 +6,5 @@
 access_token = "749661321977819138-OTMNHPCkcyxsJ7S62vLKIf3LRkY4Uz8"
 access_token_secret = "UnabuFpDvBW3ieUBJOh0Ozadp64LNn1vm3IxzBbj84l5o"
 
-# add search terms of your choice
+# add search terms of your choice (required)
```
Comment (vipulgupta2048) [inline vanilla_bot/config.py:9]: Why not `replace` required by `compulsory`

File: vanilla_bot/config.py [hunk 2]
```diff
@@ -6,5 +6,5 @@
...
 searchterms = ["hackathon"]
```
Comment (vipulgupta2048) [inline vanilla_bot/config.py:10]: dfndskjfdskfs

**Review by vipulgupta2048 — CHANGES_REQUESTED** [review]
Ahhh this doesn't look good.

**Pull request merged**